### ISSUE TITLE
Support for Insteon Remote Control Dimmer Switch, Toggle

### ISSUE THREAD

**Issue by efthymakis** [kuestess/homebridge-platform-insteonlocal#139]
I have a single Insteon Toggle Dimmer Switch that does not work with the plug-in.  
When I try to turn the light on/off, I see the following in the log:

4/26/2021, 9:59:12 PM [HOOBS CAD8@Dining Room Light@Dining Room Light@Brightness] SET handler returned write response value, though the characteristic doesn't support write response
4/26/2021, 9:59:12 PM [InsteonLocal] Setting level of Dining Room Light to 100%
4/26/2021, 10:07:28 PM [HOOBS CAD8@Dining Room Light@Dining Room Light@On] SET handler returned write response value, though the characteristic doesn't support write response
4/26/2021, 10:07:28 PM [InsteonLocal] Setting level of Dining Room Light to 0%

The light does not turn on or off.

I have no problems with the Insteon Paddle Dimmer Switches, only with the Toggle Dimmer switch (2466DW).

Insteon details:

ToggleLincTM Dimmer
INSTEON® Remote Control Dimmer Switch
(#2466Dxxx)

**Comment by efthymakis** [comment]
I should also mention the config for the device is as follows:

"name": "Dining Room Light",
            "deviceID": "3E2412",
            "deviceType": "dimmer",
            "dimmable": "yes"

I've tried defining the device as a switch and as not dimmable with no difference.

**Comment by kuestess** [comment]
@efthymakis I have an on/off toggle, but no dimmer toggle switches.  Let me see what I can find in the developer docs.

**Comment by efthymakis** [comment]
Appreciate it.  I know it's not a popular switch type.

**Comment by efthymakis** [comment]
I've been doing some simple tests at home and here are my observations:

1. The status of the light is correctly reflected in the Home app of the switch is manually pressed.  If I turn the switch on/off/dim%, the status and state are reported correctly.

2. An Off event is reflected in the Home app almost immediately, while an On event can take ~5s to be reflected in the app.

3. Turning the switch On/Off/Dim% from the Home app has no effect on the light.  The light does not turn on or off or gets set to a %.

**Comment by kuestess** [comment]
@efthymakis I haven't been able to find anything that indicates that commands for the 2466D are different than other dimmers.  Are you able to control it through the Insteon app?

**Comment by efthymakis** [comment]
Back to basics...lol

I've been using the Home app via the Insteon+ bridge for years and recently switched to Home app via Hoobs (and your plug-in) because the bridge crapped and and I fell back on the Insteon Hub2.

I haven't used the Insteon app in ages and interestingly when I opened it, I had issues with the switch when trying to control it.  The device was listed correctly, but once you try and control it, I would get an error message that the device is not reachable.

I plugged a ranger extender near the device and voila, the issue was resolved!!

Not sure why it took so long for a wiring issue to manifest itself now, but glad it was an easy fix.

I REALLY appreciate you looking into this and apologize for going down a rabbit hole.  My mind jumped to conclusions given it was the only switch in the house with a problem and a unique switch as well.
